src/pages/index.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Search from '../components/search/Search.jsx';
+import Search from '../components/search/Search';
```

Comment:
Removed `.jsx` in accordance with the [imports/extensions](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md) rule.